ansys_optical_automation/speos_process/speos_sources.py

```diff
@@ -0,0 +1,30 @@
+from ansys_optical_automation.scdm_core.base import BaseSCDM
+
+
+class Source(BaseSCDM):
+    """
+    Provides the parent class for all Speos Source types.
+
+    This class contains methods and properties that are common to all Source elements.
+    It shouldn't be used by itself. Subclasses should be used instead.
+    """
+
+    def __init__(self, name, SpeosSim, SpaceClaim):
+        super(Source, self).__init__(SpaceClaim, ["V19", "V20", "V21", "V22", "V23"])
+        self.speos_sim = SpeosSim
+        self.name = name
+        self.speos_object = None
```

Comment:
maybe add some values all sources have in comon E.g. Power, spectrum, intensity type ...